lib/rspec/core/output_wrapper.rb

```diff
@@ -0,0 +1,11 @@
+require 'delegate'
+
+class OutputWrapper < SimpleDelegator
```

Comment:
IO class adds 70 public methods. Should `output_wrapper` implement all of them? RSpec code uses only 6 methods of IO `tty? sync= closed? puts print flush`.

Also i suppose for perfomance better to implement methods without using of meta programming, like this:

```ruby
  def puts(*args)
    @output.puts(args)
  end

```